### PR TITLE
update brackets

### PR DESCRIPTION
**Pull request by BasicAlgorithm**

### CODE REVIEW

File: _docs/torneos/ab21.md [hunk 1]
```diff
@@ -18,21 +18,97 @@ tags:
...
+| Mark-Lex21          |        ***********       |
+| sospechoso          |        ***********       |
+| TheBotLess          |        ***********       |
+| thisisabot          |        ***********       |
+| TILIN               |        ***********       |
 
-*por determinarse*
+## Clasificatorias resultados 16/09/21
+
+|Equipo            | Ronda 1 | Ronda 2 | Ronda 3 | Ronda 4 | Ronda 5 | Ronda 6 | Ronda 7 | Ronda 8 | Ronda 9 | Ronda 10 | Ronda 11 | Ronda 12 | Ronda 13 | Ronda 14 | Puntaje |
```
Comment (rodp63) [inline _docs/torneos/ab21.md:52]: Please, make `Puntaje` the second column.

File: _docs/torneos/ab21.md [hunk 2]
```diff
@@ -18,21 +18,97 @@ tags:
...
+| leroyG5 |1290|			1030930|			1300|	1260|					101210|	1270|			101100|			1290|				1170|		1300|					1130|			1180|	1300|					1180| 1246924
+| TheBotLess |100970					|1100|		100000|		100581|					200100|	0|	0|					200001|			100621|					1080|	200000|				100550|				100650|	860| 1206526
+| Tilin |1300|				1300|				1300|	1190|				1300|				1240|	1300|					1031300|	1300|				1300|				1300|	1300|					1300|			1300| 1048044
+| LaUvaLoca |100151|				1010|			150|		1000|				1060|	150|				1000|				1010|	0|				980|		1090|			1150|				550|			200150|	 309464
+| BlackCat |1300|	1300|				1300|				1300|			1300|		1300				|1300|	1300|			1300|					1300|		1300|		650|			1300|				1300|	 17564
+
+	
+## Finales 18/09/21
+
+|                    Upper brackets |  |Fase 1(VO3) |\||Fase 2(VO3)|\||Fase 3(VO3)|\||Fase 4(VO3)|\||Fase 5(VO5) |\|| GANADOR
```
Comment (rodp63) [inline _docs/torneos/ab21.md:77]: Change to `BO1` and `BO3`, (best of <number>).

**Review by rodp63 — CHANGES_REQUESTED** [review]

File: _docs/torneos/ab21.md (hunk 1 above)
Comment (BasicAlgorithm) [reply _docs/torneos/ab21.md]: do it

File: _docs/torneos/ab21.md (hunk 2 above)
Comment (BasicAlgorithm) [reply _docs/torneos/ab21.md]: do it 
fix VO to BO

**Review by rodp63 — APPROVED** [review]

**Pull request merged**